docs/user-guide/dream/dream-visualize-absorption.ipynb

```diff
@@ -0,0 +1,227 @@
+    "\n",
+    "We load a dataset from a Geant4 simulation only to get access to the detector geometry."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dg = dream.io.load_geant4_csv(dream.data.get_path(\"data_dream0_new_hkl_Si_pwd.csv.zip\"))[\"instrument\"]\n",
+    "dg = sc.DataGroup({key: detector[\"events\"] for key, detector in dg.items()})\n",
+    "dg.keys()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4",
+   "metadata": {},
+   "source": [
+    "## Inhomogeneity from absorption in mantle detector\n",
+    "\n",
+    "If the sample absorbs or scatters a large fraction of the incoming neutrons, the intensity in the detector will vary even if the scattering is inhomogeneous.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5",
+   "metadata": {},
+   "source": [
+    "### Rod-like sample oriented along x-axis"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# The material determines the rate of absorption and scattering\n",
+    "vanadium =  Material(\n",
+    "    scattering_params=ScatteringParams.for_isotope('V'),\n",
+    "    effective_sample_number_density=sc.scalar(0.07192, unit='1/angstrom**3')\n",
+    ")\n",
+    "# The shape determines the shape and the orientation of the sample\n",
+    "rod_shape = Cylinder(\n",
+    "    radius=sc.scalar(1, unit='cm'),\n",
+    "    height=(height := sc.scalar(5., unit='cm')),\n",
+    "    # Cylinder is oriented along the x-axis\n",
+    "    symmetry_line=(symmetry_line := sc.vector([1, 0, 0])),\n",
+    "    center_of_base=-height * symmetry_line / 2,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "transmission_fraction_mantle = compute_transmission_map(\n",
+    "    rod_shape,\n",
+    "    vanadium,\n",
+    "    beam_direction=sc.vector([0, 0, 1]),\n",
+    "    wavelength=sc.linspace('wavelength', 4, 8, 20, unit='angstrom'),\n",
+    "    # To make it faster, don't compute the transmission fraction for every detector pixel, it's not necessary for the visualization.\n",
+    "    detector_position=dg['mantle'].coords['position']['strip', ::4]['wire', ::2].copy(),\n",
+    "    quadrature_kind='cheap',\n",
+    ")\n",
+    "\n",
+    "# The visualization expects the `position` coord to denote detector positions, and `wavelength` to be the last dimension.\n",
+    "transmission_fraction_mantle.coords['position'] = transmission_fraction_mantle.coords.pop('detector_position')\n",
+    "dream.instrument_view(transmission_fraction_mantle.transpose((*set(transmission_fraction_mantle.dims) - {'wavelength'}, 'wavelength')), dim='wavelength', camera=Camera(position=sc.vector((-2, 2, -2), unit='m'), look_at=sc.vector((1, -1/3, 1/2), unit='m')))"
```

Comment:
```suggestion
    "dream.instrument_view(transmission_fraction_mantle, dim='wavelength', pixel_size=20.0)"
```
Not sure setting the camera is so important, since we are also able to move the camera around in the online docs.
It adds a lot of noise to the notebook, which can scare away some users?